### ISSUE TITLE
TYPE / EXISTS do not work as expected for indexes

### ISSUE THREAD

**Issue by stockholmux** [RedisLabsModules/RediSearch#109]
Drunken monkey testing continues...

Redis 4.0 GA / RediSearch 0.19.3

```
127.0.0.1:6379> FT.CREATE anIndex SCHEMA something TEXT
OK
127.0.0.1:6379> TYPE anIndex
none
127.0.0.1:6379> EXISTS anIndex
(integer) 0
127.0.0.1:6379> FT.INFO anIndex
 1) index_name
 2) anIndex
 3) fields
 4) 1) 1) something
       2) type
       3) TEXT
       4) weight
       5) "1"
*...more / irrelevant stuff*
```

After tinkering I figured out it is because of the automatic 'idx' prefixing (`> type idx:anIndex` works for example), but it's really an odd behaviour and counterintuitive compared to all in-built Redis commands and even `ft.sug*` commands also work as expected with `type` / `exists`. Two ideas to make this behaviour straight-forward:
1) ability to supply an alternate prefix
2) `FT.CREATE` could return the index key instead of just OK

Take of it what you will.

**Comment by dvirsky** [comment]
The second one is an interesting idea, however, in cluster mode even that is not feasible as each shard has a different version of the key name. What can I say, RediSearch is not your regular redis extension. It's best viewed as an atomic unit built on top of redis, that fiddling with using ordinary redis tools and methods leads to a world of pain :)

I'm keeping it open for now as a reminder to give it some extra thought. We can continue the discussion and other people are invited to chime in. :)

**Comment by abrookins** [comment]
Bump! This would be useful to avoid having to do things like e.g. run FT.INFO and check for error response to see if an index exists.

**Comment by ashtul** [comment]
RediSearch 2.0 has FT._LIST which returns all the indexes names.

**Comment by abrookins** [comment]
That’s true - can we make this a not hidden command? I suspect users would use it.
